### PR TITLE
Add autohotkey

### PR DESCRIPTION
**Pull request by noahgaertner**
Adds Autohotkey to the Package repository.
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/882)

### CODE REVIEW

File: manifests/Lexikos/AutoHotKey/1.1.32.00.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Id: lexikos.autohotkey
```
Comment (MarNicGit) [inline manifests/Lexikos/AutoHotKey/1.1.32.00.yaml:1]: The package ID should be `Lexikos.AutoHotkey`. Your folder name should be `AutoHotkey` as well, following AHK's own casing.

File: manifests/Lexikos/AutoHotKey/1.1.32.00.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Version: 1.1.32.00
+Publisher: Lexikos
+Name: AutoHotKey
```
Comment (MarNicGit) [inline manifests/Lexikos/AutoHotKey/1.1.32.00.yaml:4]: Official AHK website names it `AutoHotkey`, without a capital 'K'

File: manifests/Lexikos/AutoHotKey/1.1.32.00.yaml (hunk 1 above)
Comment (noahgaertner) [reply manifests/Lexikos/AutoHotKey/1.1.32.00.yaml]: Fixed in new commit, just pushed and waiting on Pipelines

File: manifests/Lexikos/AutoHotKey/1.1.32.00.yaml (hunk 2 above)
Comment (noahgaertner) [reply manifests/Lexikos/AutoHotKey/1.1.32.00.yaml]: Fixed in new commit as well

File: manifests/Lexikos/AutoHotKey/1.1.32.00.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+LicenseUrl: https://github.com/Lexikos/AutoHotkey_L/blob/master/license.txt
+AppMoniker: AHK
+Description: AutoHotkey is a free, open source macro-creation and automation software utility that allows users to automate repetitive tasks. It is driven by a custom scripting language that is aimed specifically at providing keyboard shortcuts, otherwise known as hotkeys.
+Homepage: https://autohotkey.com/
+Installers:
+  - Arch: x86
+    Url: https://github.com/Lexikos/AutoHotkey_L/releases/download/v1.1.32.00/AutoHotkey_1.1.32.00_setup.exe
+    Sha256: CE505D272F8D36C5599AC81F005E1D2D586EEAAA778C779AD858E44FDADFDB0D
+    InstallerType: EXE
+
```
Comment (izep) [inline manifests/Lexikos/AutoHotKey/1.1.32.00.yaml:15]: For a silent install, after InstallerType add:
```
    Switches:
      Silent: /S
```

**Pull request closed without merging**